ex04.ex

```diff
@@ -55,8 +57,14 @@ defmodule Ex04 do
 
   """
 
-  def min . . . "your code"
-
+  def min(list) do
```

Comment:
## -1

This is good, but personally I like to see the functions split out a little:

~~~ elixir
  def min(list), do: reduce(list, &min_of/2)
  def min_of(a, b) when a < b, do: a
  def min_of(a, b), do: b
~~~